databricks/koalas/tests/test_dataframe.py

```diff
@@ -760,6 +760,56 @@ def test_xs(self):
         with self.assertRaisesRegex(KeyError, msg):
             kdf.xs(('mammal', 'dog', 'walks', 'foo'))
 
+    def test_where(self):
+        pdf1 = pd.DataFrame({'A': [0, 1, 2, 3, 4], 'B': [100, 200, 300, 400, 500]})
+        pdf2 = pd.DataFrame({'A': [0, -1, -2, -3, -4], 'B': [-100, -200, -300, -400, -500]})
+        kdf1 = ks.from_pandas(pdf1)
+        kdf2 = ks.from_pandas(pdf2)
+
+        set_option("compute.ops_on_diff_frames", True)
```

Comment:
I think we can add these tests at `OpsOnDiffFramesEnabledTest`

Reply:
yeah i think so too.